Gemfile

```diff
@@ -27,6 +27,6 @@ else
   gem 'puppet', :require => false
 end
 
-gem 'hocon', '~> 0.9.3',       :require => false
```

Comment:
It all depends on "your" control and commitment to keep the hocon API compatible. Usually, a `~> 1.0` would be a safe choice, or `~> 1.2`, if you ar depending on features from the 1.2.x series.

If you need to run tests against multiple versions of the underlying gem, you might use the `location_from_env` function, and expand the travis/jenkins matrices with pins to more specific versions, e.g. `~>1.0.0` through `~>1.2.0`...